src/utils/atomFamily.ts

```diff
@@ -80,5 +90,18 @@ export function atomFamily<Param, Value, Update>(
       atoms.splice(index, 1)
     }
   }
+  createAtom.gc = (fn: ShouldRemove<Param> | null) => {
+    shouldRemove = fn
+    if (!shouldRemove) return
+    let index = 0
+    while (index < atom.length) {
+      const item = atoms[index]
+      if (shouldRemove(item[2], item[0])) {
+        atoms.splice(index, 1)
+      } else {
+        ++index
+      }
+    }
```

Comment:
So, it sets `shoudRemove` and check all atoms just once.
Is it clearer if we expose two functions? set and run?